workspaces/jenkins/plugins/jenkins-backend/README.md

```diff
@@ -120,6 +121,7 @@ metadata:
     'jenkins.io/job-full-name': teamA/artistLookup-build
 ```
 
+The projectCountLimit is optional and if not set, the default limit is 50.
```

Comment:
```suggestion
The `projectCountLimit` is optional and if not set, the default limit is 50.
```

Reply:
Done in [e68de0c](https://github.com/backstage/community-plugins/pull/796/commits/e68de0c948a2611e123622f9f1b03ea478469473)